tests/system/model_monitoring/test_model_monitoring.py

```diff
@@ -246,16 +248,23 @@ class TestBasicModelMonitoring(TestMLRunSystem):
     # Set image to "<repo>/mlrun:<tag>" for local testing
     image: Optional[str] = None
 
-    @pytest.mark.timeout(270)
-    def test_basic_model_monitoring(self) -> None:
+    @pytest.mark.timeout(540)
+    @pytest.mark.parametrize("with_sql_target", [True, False])
+    def test_basic_model_monitoring(self, with_sql_target: bool) -> None:
```

Comment:
Does it pass now? I remember it failed. If you fixed it here - awesome!
Was it only the timeout?

Reply:
it passes :) didn't change anything in the logic. I raised the timeout 2x because we are running it "twice", actually I wasn't sure if pytest timeout resets on each pytest param running.

Reply:
The timeout applies to each test item (params set) separately.
But since it solves the issue, let's keep it as is.